### PR TITLE
#122 - Using Jacoco plugin configuration from parent pom

### LINKED ISSUE #122: Use Jacoco plugin settings from parent pom
`ppom`  verion `0.3.3` now contains reference Jacoco plugin configuration, it should be used in the projec

### PR DESCRIPTION
**Pull request by olegmoz**
Closes #122

### CODE REVIEW

**Comment by 0crat** [comment]
This pull request [#124](https://github.com/artipie/docker-adapter/issues/124) is assigned to @paulodamaso[/z](https://www.0crat.com/u/paulodamaso), here is [why](https://www.0crat.com/footprint/CT2E6TK9B/ac2aef18-c605-4e6c-93f2-c424487c9728); the budget is 15 minutes, see [§4](http://www.zerocracy.com/policy.html#4); please, read [§27](http://www.zerocracy.com/policy.html#27) and when you decide to accept the changes, inform @g4s8[/z](https://www.0crat.com/u/g4s8) (the architect) right in this ticket; if you decide that this PR should not be accepted ever, also inform the architect; this [blog post](http://www.yegor256.com/2015/02/09/serious-code-reviewer.html) will help you understand what is expected from a code reviewer; there will be a monetary reward for this job

File: pom.xml [hunk 1]
```diff
@@ -148,81 +148,6 @@ SOFTWARE.
...
-          <execution>
-            <id>jacoco-check</id>
-            <phase>test</phase>
-            <goals>
-              <goal>check</goal>
-            </goals>
-            <configuration>
-              <rules>
-                <rule>
-                  <element>BUNDLE</element>
```
Comment (paulodamaso) [inline pom.xml:174]: @olegmoz I don't think that's a good idea to use generic limits defined in parent pom. I think that each repository should have its own coverage limits

Comment (olegmoz) [reply pom.xml:174]: @paulodamaso actually there are no limits specified in parent and it's one of the points - code coverage becomes a report and not a mandatory check for each and every commit

**Comment by olegmoz** [comment]
@paulodamaso ping

**Comment by g4s8** [comment]
@0crat assign me

**Comment by 0crat** [comment]
> @0crat assign me ([here](https://github.com/artipie/docker-adapter/issues/124#issuecomment-632641415))

@g4s8 This pull request [#124](https://github.com/artipie/docker-adapter/issues/124) is assigned to @g4s8[/z](https://www.0crat.com/u/g4s8), here is [why](https://www.0crat.com/footprint/CT2E6TK9B/38005c78-8dec-4abe-a526-58860bf717a4); the budget is 15 minutes, see [§4](http://www.zerocracy.com/policy.html#4); please, read [§27](http://www.zerocracy.com/policy.html#27) and when you decide to accept the changes, inform @g4s8[/z](https://www.0crat.com/u/g4s8) (the architect) right in this ticket; if you decide that this PR should not be accepted ever, also inform the architect; this [blog post](http://www.yegor256.com/2015/02/09/serious-code-reviewer.html) will help you understand what is expected from a code reviewer; there will be no monetary reward for this job

**Comment by 0crat** [comment]
Job was finished in 0 hours, bonus for fast delivery is possible (see [§36](http://www.zerocracy.com/policy.html#36))

**Pull request merged**